### ISSUE TITLE
Add Fedora to the TODO

### ISSUE THREAD

**Issue by rpgdev** [Bercik1337/rt-auto-install#23]
Would love to have Fedora support on this.

**Comment by Bercik1337** [comment]
Hey @rpgdev thanks for posting.
Unfortunately fedora is not .deb based distribution. To modify script it's not just a matter of changing APT to YUM, a lot of things needs to be changed. A LOT.
I'm not saying no, it's just don't expect this to happen anytime soon. Unless someone will participate, I simply don't have time to rediscover alternate distribution tree.

I'm marking this as help wanted, so maybe we'll find helping hands. Else, don't expect this to happen sooner than december.

**Comment by rpgdev** [comment]
@Bercik1337 fair enough, thank you for taking the time to explain what stands in the way of addressing it.

**Comment by Pehaspe** [comment]
Hello, is there any progress on the subject?

**Comment by Bercik1337** [comment]
No. I don't see any chance of this happening unless someone will help out. I don't have time to re-engineer it myself. Sorry
